nni/tools/nnictl/launcher.py

```diff
@@ -574,3 +574,28 @@ def view_experiment(args):
 def resume_experiment(args):
     '''resume an experiment'''
     manage_stopped_experiment(args, 'resume')
+
+def view_external_experiment(args):
+    '''view a experiment from external path'''
+    # validate arguments
+    if not os.path.exists(args.experiment_dir):
+        print_error('Folder %s does not exist!' % args.experiment_dir)
+        exit(1)
+    if not os.path.isdir(args.experiment_dir):
+        print_error('Path %s is not folder directory!' % args.experiment_dir)
+        exit(1)
+    experiment_id = os.path.basename(args.experiment_dir)
```

Comment:
This might not work for `~/nni/foo/` (ends with slash)
And since the experiment ID is not included in logDir / experimentWorkingDirectory, I think the argument is a little stange.

Reply:
add validation logic.

Reply:
`nnictl view -e /home/lz/nni-experiments/Su83qgOR/` cannot pass validation.
I think it's frustrating since bash will automatically add the tail space with tab completion.
Suggest use `Path(experiment_dir).name` to replace basename.

Reply:
sure, updated to `Path(args.experiment_dir).name`.